contributed_definitions/nyaml/NXmpes.yaml

```diff
@@ -363,7 +397,15 @@ NXmpes(NXobject):
           varied axis may be for example energy, momentum, spatial coordinate, pump-probe
           delay, spin index, temperature, etc. The axes traces should be linked to the
           actual encoder position in NXinstrument or calibrated axes in NXprocess.
-
+      energy(NX_NUMBER):
```

Comment:
There should be at least one energy axis, either of type binding or kinetic energy. Are there applications of MPES were this is not the case?

Reply:
PEEM (#65) could potentially be measured without an energy axis.

Reply:
Hm right, then it cannot be required. Anyway, as discussed yesterday, we need to talk about this further, since at the moment, basically any data can be stored in NXmpes. We want to somehow enumerate which axes are to be expected in order for an experiment to be called MPES.

Reply:
I mean we can also say that PEEM is not part of NXmpes if it does not require an energy axis (i.e. is it still a photoemission _spectroscopy_ then!?) or we introduce the requirement further down the chain, e.g., building an NXmpes_energy_resolved (not very fond of the naming, though) in between (or above, like NXphotoemission -> NXmpes; NXphotoemission -> NXpeem). We can also give some axis but set them to recommended/optional so users might pick what they need. This would at least define the concepts we are using.

Reply:
Yeah, if we say we are talking about _spectroscopy_, then PEEM probably doesn't fit nicely. But I agree, the techniques are so similar that maybe an overarching application like NXphotomission could be useful, with NXmpes requiring an energy axis. But the question remains, what data fields do we require so it is an NXphotoemission experiment?

For know, I have made the energy axis recommended.